postreise/plot/plot_bar_generation_max_min_actual.py

```diff
@@ -32,10 +34,37 @@ def plot_bar_generation_max_min_actual(
     :param int/float fontsize: font size of the texts shown on the plot.
     :param plot_show: show the plot or not, defaults to True.
     :return: (*matplotlib.axes.Axes*) -- axes object of the plot.
+    :raises TypeError:
+        if ``interconnect`` and ``gen_type`` are not str
+        if ``fontsize`` is not an integer or a float and/or
+    :raises ValueError: if ``interconnect is not valid
     """
-    grid = scenario.state.get_grid()
+    _check_scenario_is_in_analyze_state(scenario)
```

Comment:
I think we did this check in `sum_generation_by_type_zone` which is called below.

Reply:
Discarded